tests/e2e/steps_consumer_misbehaviour.go

```diff
@@ -173,7 +172,7 @@ func stepsStartChainsWithSoftOptOut(consumerName string) []Step {
 				ChainID(consumerName): ChainState{
 					ValPowers: &map[ValidatorID]uint{
 						ValidatorID("alice"): 500,
-						ValidatorID("bob"):   20,
+						ValidatorID("bob"):   0,
```

Comment:
Is it expected? Bob shouldn't be in the consunmer's valset even with 0 vp, right?